src/thor/trace_attributes_action.cc

```diff
@@ -428,8 +428,8 @@ json::MapPtr thor_worker_t::trace_attributes(
         try {
           trip_path = route_match(controller);
           if (trip_path.node().size() == 0)
-            throw;
-        } catch (...) {
```

Comment:
i had previously changed these because they were very oddly catching only `valhalla_exception_t` but it turns out they only thing they really throw on purpose is `std::runtime_error`, the reason they weren't catching `...` before was because they wanted the interrupt to remain uncaught, so i had to revert that and catch a generic exception here to allow custom interrupts to get through. what this means is that interrupts work most cleanly when they throw something that isnt decended from `std::exception`